packages/graphql_parser/graphql_parser.0.9.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Andreas Garnaes <andreas.garnaes@gmail.com>"
+authors: "Andreas Garnaes <andreas.garnaes@gmail.com>"
+homepage: "https://github.com/andreas/ocaml-graphql-server"
+doc: "https://andreas.github.io/ocaml-graphql-server/"
+bug-reports: "https://github.com/andreas/ocaml-graphql-server/issues"
+dev-repo: "git+https://github.com/andreas/ocaml-graphql-server.git"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
+  "menhir" {build}
+  "alcotest" {with-test & >= "0.8.1"}
+  "result"
```

Comment:
the dependency on `result` should not be needed, since you require `ocaml >=4.03.0` anyways (and result is a backwards compatibility package for ocaml <4.03.0). NB: this may require changes to code / build system, so no need to modify it for this release, but you may want to drop it in future releases.